### ISSUE TITLE
public library / stable ABI

### ISSUE THREAD

**Issue by umlaeute** [CICM/CicmWrapper#14]
currently CicmWrapper seems to be included verbatim into the libraries that use it (e.g. via `git submodules`).

if the API/ABI is stable: would it make sense to install a shared-library version of the wrapper into a public place (e.g. `/usr/lib/libCicmWrapper.so.0`) and dylink the externals?

**Comment by pierreguillot** [comment]
Yes, you're right ! It would be better. I have to update the library first, then I'll try this.
